app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for person in people:
+        person_list.append(Person(person["name"], person["age"]))
+
+    for person in people:
+        current_person = Person.people[person["name"]]
+        if "wife" in person and person["wife"]:
+            current_person.wife = Person.people[person["wife"]]
+        elif "husband" in person and person["husband"]:
```

Comment:
Instead of explicitly checking for the presence of a key in the dictionary, use the `dict.get()` method. This approach simplifies the code and avoids potential `KeyError` exceptions .